src/main/java/com/cryptomorin/xseries/skull/XSkull.java

```diff
@@ -424,6 +610,12 @@ private static void cacheProfile(GameProfile profile) {
         }
```

Comment:
I've noticed you're using a lot of LOGGER.debug, you should just change them all to LOGGER.error that's definitely more suitable.